.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!-- Thank you for contributing! -->
+
+### Before submitting the PR, please make sure you do the following
+
+- [ ] Read the [contributing guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md).
+- [ ] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
```

Comment:
As per discussion with @GrygrFlzr in Discord, we should directly link to the PR Guidelines section https://github.com/vitejs/vite/blob/main/.github/contributing.md#pull-request-guidelines
I think it is useful to have both 

- [ ] Read the [contributing guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md).
- [ ] Read the [Pull Request guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md#pull-request-guidelines) and follow the [Commit Convention](https://github.com/vitejs/vite/blob/main/.github/commit-convention.md).